.npmignore

```diff
@@ -0,0 +1,53 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency files and directories
+node_modules
+jspm_packages
+yarn.lock
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+# mac files
+.DS_Store
+
+# vim swap files
+*.swp
+
+package-lock.json
+
+# project specific
+*.tgz
+tap-snapshots
+test
+.travis.yml
+var
```

Comment:
Can you remove this file?

Reply:
yes, I can, but, I wanted `static` to be ignored by git while be included in the npm. I see it's only possible with adding `.npmignore`.

plus I think that it's not really needed to include tests and other stuff into the package itself.

Reply:
Oh right. Ok.